op/op-social-name/src/index.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+import { type socialOperatorT, focusRole } from 'frog-utils';
+
+const meta = {
+  name: 'Assign group by name',
+  shortDesc: 'Maps students to groups by name',
+  description:
+    'Input a list of student names, and group names, and it matches the students to the groups'
+};
+
+const config = {
+  type: 'object',
+  required: ['groupnames', 'defaultgroup', 'studentmapping'],
+  properties: {
+    groupnames: {
```

Comment:
I would suggest to change the naming of the config fields to be consistent with the rest of the Engine. I believe:
- `groupnames` should be `groupingKeys`. This one is particularly confusing as groupnames sounds like it would be groupingValues, while it is groupingKeys.
- `defaultgroup` should be `defaultGroupingValues`